gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/RepositoryLoader.java

```diff
@@ -36,7 +36,7 @@
 import org.slf4j.LoggerFactory;
 
 public final class RepositoryLoader {
-    private static final Logger log = LoggerFactory.getLogger(RepositoryExplorer.class);
+    private static final Logger log = LoggerFactory.getLogger(RepositoryLoader.class);
```

Comment:
you should use a live template for this in intellij! I imagine this was a copy paste thing

Reply:
<img width="1192" alt="Screenshot 2024-10-15 at 12 54 44" src="https://github.com/user-attachments/assets/8856c3ab-3bde-4a07-ad51-aff67e58c05d">

Reply:
thats so cool thank!